### PR TITLE
feat: allow querying by name using route endpoints/query/:token_or_name

### PR DESCRIPTION
**Pull request by Ziinc**
allows querying by name via the `/api/endpoints/query/:token_or_name` route, to make routing simpler.
[ticket](https://www.notion.so/supabase/Update-endpoints-query-route-to-handle-names-378b08fac27641d8b38277c540a3e53d?pvs=4)

### CODE REVIEW

File: test/logflare/utils_test.exs [hunk 1]
```diff
@@ -3,4 +3,6 @@ defmodule Logflare.UtilsTest do
   use ExUnit.Case, async: true
   import Logflare.EnumDeepUpdate
   doctest Logflare.EnumDeepUpdate
+  import Logflare.Utils
+  doctest Logflare.Utils
```
Comment (filipecabaco) [inline test/logflare/utils_test.exs:7]: I don't think we need this, we're testing the dependency

File: lib/logflare/utils.ex [hunk 2]
```diff
@@ -0,0 +1,16 @@
+defmodule Logflare.Utils do
```
Comment (filipecabaco) [inline lib/logflare/utils.ex:1]: We're only using this in one piece of the code and it's calling a library directly, I don't see the need to have a utils library for this.

**Review by filipecabaco — APPROVED** [review]
Looks good but I don't see the need for the Logflare.Utils at this point so I would remove it.

Wi still approve as this is a minor thing that should not block you IMO

**Comment by Ziinc** [comment]
Removed the Utils module and made the check a private function 👍

**Pull request merged**